Library/Homebrew/formula.rb

```diff
@@ -2130,6 +2130,7 @@ def to_hash
         "used_options"            => tab.used_options.as_flags,
         "built_as_bottle"         => tab.built_as_bottle,
         "poured_from_bottle"      => tab.poured_from_bottle,
+        "time"                    => Time.at(tab.time).strftime("%Y-%m-%d %H:%M:%S"),
```

Comment:
What does just 
```suggestion
        "time"                    => tab.time,
```

output?

Wondering if a dedicated method might be nicer for this format.

Reply:
I adapted that from [Tab.to_s](https://github.com/Homebrew/brew/blob/76b87c4a00ed636bcf59c104b85863637609b66d/Library/Homebrew/tab.rb#L391) ([which is where `brew info` gets its time](https://github.com/Homebrew/brew/blob/76b87c4a00ed636bcf59c104b85863637609b66d/Library/Homebrew/cmd/info.rb#L323-L324)).

`tab.time` is the unix epoch, which for my purposes, is slightly trickier but good enough. What would you consider most standard/user-friendly/etc.? Here are some options:

```ruby
"time" => Time.at(tab.time).strftime("%Y-%m-%d %H:%M:%S"),    # "time": "2022-08-29 17:29:30",
```
```ruby
"time" => tab.time,                                           # "epoch": 1661819370,
```
```ruby
"epoch" => tab.time,                                          # "epoch": 1661819370,
"time"  => Time.at(tab.time).strftime("%Y-%m-%d %H:%M:%S"),   # "time": "2022-08-29 17:29:30",
```

```ruby
"time" => {                                                   # "time": {
  "epoch" => tab.time,                                        #   "epoch": 1661819370,
  "local" => Time.at(tab.time).strftime("%Y-%m-%d %H:%M:%S"), #   "local": "2022-08-29 17:29:30"
},                                                            # },
```

If we include local time here, we could certainly have a `Time.at(tab.time).strftime()` wrapper as a tab method, since it'll be used in two places.

Reply:
@jacktose What would you think about adding an extra method to `tab.rb` to be something like `pretty_time` so that you could call `tab.pretty_time` both here and in `Tab#to_s`?

Reply:
A bit partial to having `tab.time` here, since that can be more reliably transformed into whatever format the reader needs. The `strftime` transformation likely depends on the locale.

Reply:
I'm fine either way so let's just do the `time => tab.time` outputting the epoch 👍🏻